src/components/TodoComponents/Todo.css

```diff
@@ -63,8 +63,8 @@ input {
 .button {
     border: 2px solid white;
     border-radius: 6px;
-    background-color: skyblue;
-    color: gray;
+    background-color: #F27B21;
+    color: white;
```

Comment:
try to stick to one convention for your colors. for instance. either names like white or hex codes like #fff; we all do this at times where we mix semantics but it is bad practice and should be avoided for clarity